roles/php/tasks/main.yml

```diff
@@ -1,84 +1,36 @@
 ---
-- name: Install PHP
+- name: Check that default PHP version is supported
+  fail:
+    msg: "Default PHP version {{ php_version }} is not supported"
+  when: php_version not in php_supported_versions
+
+- name: Uninstall unsupported PHP versions
   homebrew:
-    name: "php@{{ php_version }}"
-    state: latest
+    name: "php@{{ item }}"
+    state: absent
+  with_items: "{{ php_unsupported_versions }}"
+
+- name: Stop all supported PHP-FPM
+  command: "brew services stop php@{{ item }}"
+  with_items: "{{ php_supported_versions }}"
```

Comment:
Maybe it is not a real problem, but there is a fail when trying to stop php fpm of previously not installed versions of php.
I don't know if this can cause issues or not

<img width="407" alt="Capture d’écran 2019-12-19 à 08 58 02" src="https://user-images.githubusercontent.com/11588007/71155343-c3b52700-223d-11ea-8028-5996235bb8a6.png">